pilot/pkg/dns/dns.go

```diff
@@ -153,6 +156,7 @@ func (h *LocalDNSServer) UpdateLookupTable(nt *nds.NameTable) {
 		lookupTable.buildDNSAnswers(altHosts, ipv4, ipv6, h.searchNamespaces)
 	}
 	h.lookupTable.Store(lookupTable)
+	h.initialized = true
```

Comment:
Is this concurrency safe?

Reply:
UpdateLookUpTable is not concurrent right?

Reply:
I was thinking it was but I suppose not. But UpdateLookUpTable may change the value while the probe checks it?